helm/templates/standalone/statefulset-standalone.yaml

```diff
@@ -119,6 +137,11 @@ spec:
         - name: config
           configMap:
             name: {{ template "mongodb.fullname" . }}
+        {{- if .Values.usePassword }}
+        - name: init-scripts-config
+          configMap:
+            name: {{ template "mongodb.fullname" . }}-init-scripts
```

Comment:
Can the init-scripts flag be named accordingly (instead of using the `usePassword` flag for `initi-scripts`?